integration/pod_hostname_test.go

```diff
@@ -49,16 +50,21 @@ func TestPodHostname(t *testing.T) {
 				WithHostNetwork,
 			},
 			expectedHostname: hostname,
+			skipOnWindows:    true,
 		},
 		"host network pod with custom hostname should fail": {
 			opts: []PodSandboxOpts{
 				WithHostNetwork,
 				WithPodHostname("test-hostname"),
 			},
-			expectErr: true,
+			expectErr:     true,
+			skipOnWindows: true,
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
+			if test.skipOnWindows && goruntime.GOOS == "windows" {
```

Comment:
Yes. We are skipping the ``host network pod without custom hostname`` and ``host network pod with custom hostname should fail`` tests because they require host network, but we are still running the test ``regular pod with custom hostname``.

Reply:
Done